var/spack/repos/builtin/packages/xrootd/package.py

```diff
@@ -123,6 +123,13 @@ class Xrootd(CMakePackage):
     conflicts("openssl@3:", when="@:5.3.99")
 
     extends("python", when="+python")
+
+    # https://github.com/spack/spack/pull/38547
```

Comment:
What's the reason for adding the link to this PR? Don't you want to add the link to the XRootD PR?

Reply:
Ah indeed wrong URL, I've changed the comment